### DISCUSSION TITLE
ERRO - TypeScript não está carregando

### DISCUSSION THREAD

**Discussion by Al3xFlores** [frontendbr/forum#1668] [Perguntas e Respostas] [answered]
**Olá! 
Estou estudando TypeScript para posterior passar a estudar angular, porem no primeiro teste apareceu o seguinte erro;** 
O arquivo C:\Users\alexm\AppData\Roaming\npm\tsc.ps1 não pode ser carregado porque a execução de scripts foi desabilitada neste 
sistema. Para obter mais informações, consulte about_Execution_Policies em https://go.microsoft.com/fwlink/?LinkID=135170.
No linha:1 caractere:1
+ tsc type.ts
+ ~~~
    + CategoryInfo          : ErrodeSegurança: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess

Alguém já teve esse problema ou tem conhecimento de como resolver?

**Comment by wilsonneto-swe** [comment]
Acredito que se você executar "Set-ExecutionPolicy AllSigned" vai resolver.

O problema ai é políticas de acesso do powershell, nem chegou na parte do Typescript pra dar o erro ai...

Ou, executa pelo cmd/prompt mesmo.

Depois posta se ajudou, abraços!

**Comment by Al3xFlores** [comment]
Bom dia Wilsonneto, obrigado por respondeu meu tópico. 
Esse comando  "Set-ExecutionPolicy AllSigned" não está sendo possível executar no CMD

**Reply by luandradesp** [reply]
`Set-ExecutionPolicy Unrestricted`

**Reply by MaurilioSemeao** [reply]
No cmd não vai executar, tem que abrir o powerShell  com permissão de adm  e executar a linha de comando

**Comment by Al3xFlores** [comment]
Wilsonneto encontrei a solução em um vide-o, vou estar compartilhando para quem tiver o mesmo problema; https://www.youtube.com/watch?v=J30n3lMyvbY

**Reply by ViniciussAlencar** [reply]
Obrigado meu amigo, resolvi o problema!

**Reply by FabioHA** [reply]
Muito Obrigado  Al3xFlores!!! Esse vídeo resolveu meu problema.

**Reply by ilessalobo** [reply]
Obrigada me salvou!

**Reply by ciceroduvall23** [reply]
Valeu amigo,me salvou

**Reply by CarlosBaiao** [reply]
obrigadooooo

**Reply by PiGuiMc** [reply]
obrigado!!! Ajudou aqui!

**Reply by VValdemiro** [reply]
Obrigado

**Reply by AndersonHelder93** [reply]
Muito Obrigado, eu já estava desistindo até ver essa solução, me ajudou muito

**Reply by mcstafilho** [reply]
Obrigado!!

**Reply by TiagoPiccinatto** [reply]
valeu !!

**Reply by LuizReisDev** [reply]
Muito obrigado. Eu também consegui após executar este procedimento do vídeo!

**Reply by ferreira28ian** [reply]
vlw amigão

**Comment by luandradesp** [comment, marked as answer]
Solução do vídeo
**Se ao digitar**
` Get-ExecutionPolicy`
resultar em: `Restricted`

**Solução:**
`Set-ExecutionPolicy Unrestricted`

Checar se ao digitar:
```
Get-ExecutionPolicy
``
retorna 
Unrestricted
```
:)

**Reply by wesleyalvesdeveloper99** [reply]
Oi se alguém estiver ainda com esse problema só executar esse comando aqui:
OBS: Tem que estar como administrador no PowerShell
Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass

**Reply by JuuJxp** [reply]
Muito obrigada! funcionou!

**Reply by braiansmarzaro** [reply]
A resposta do @wesley44354 funcionou! Obrigado!

**Reply by caiquefrd** [reply]
funcionou perfeito, brigadão!

**Comment by VictorCiechovicz** [comment]
https://www.youtube.com/watch?v=mmXzYxR7s9c

**Comment by leandrogarciadev** [comment]
muito obrigado, funcionou

**Comment by Cristhyam-Augusto** [comment]
Estava com o mesmo erro, obrigado pela ajuda, funcionou.
